packages/contracts/test/20_integration-testing/21_deployment.ts

```diff
@@ -88,26 +90,35 @@ type FixtureResult = {
   deployer: SignerWithAddress;
   pluginRepo: PluginRepo;
   pluginRepoRegistry: PluginRepoRegistry;
+  managementDaoProxy: DAO;
 };
 
 async function fixture(): Promise<FixtureResult> {
   // Deploy all
   const tags = ['CreateRepo', 'NewVersion'];
-  await deployments.fixture(tags);
 
+  await deployments.fixture(tags);
   const [deployer] = await ethers.getSigners();
 
+  // Plugin repo
+  const {pluginRepo, ensDomain} = await findPluginRepo(env);
+  if (pluginRepo === null) {
+    throw `PluginRepo '${ensDomain}' does not exist yet.`;
+  }
+
   // Plugin repo registry
   const pluginRepoRegistry = PluginRepoRegistry__factory.connect(
     getLatestNetworkDeployment(getNetworkNameByAlias(productionNetworkName)!)!
       .PluginRepoRegistryProxy.address,
     deployer
   );
 
-  const {pluginRepo, ensDomain} = await findPluginRepo(env);
-  if (pluginRepo === null) {
-    throw `PluginRepo '${ensDomain}' does not exist yet.`;
-  }
+  // Management DAO proxy
+  const managementDaoProxy = DAO__factory.connect(
+    getLatestNetworkDeployment(getNetworkNameByAlias(productionNetworkName)!)!
```

Comment:
I think better to throw an error if the network is unsupported rather than use `!` twice.

Reply:
Fixed everywhere in cf903c14dfc8c3290556685096d7cb8225dbf7da and created [task OS-1084](https://aragonassociation.atlassian.net/browse/OS-1084.) to fix it in the template.